.github/github-actions.yml

```diff
@@ -0,0 +1,17 @@
+name: Issue assignment
+
+on:
+    issues:
+        types: [opened]
+        comment: 
+          - 'self-assign'

```

Comment:
It should assign when new issue comment with `#self-assign` or `#take` is created

Reply:
I am trying to parse through the comment using the filter funcntionaility like ```#self-assign in:comment ```, but i don't if it works. I also tried using other workflows in my own repo but i am unable to automatically  assign issues. Can you point out some resources It will help a lot.

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#available-events

Reply:
can i consider ```#self-assign``` and ```take``` as labels,  otherwise I don't know ways to make sure that the these words in comments. can you suggest a way to check  if these words are in the comments

Reply:
using as labels is unacceptable. Take a look here [issue_comment](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment).